template/layout/default.spec.ts

```diff
@@ -0,0 +1,8 @@
+// Import { shallowMount } from '@vue/test-utils';
```

Comment:
Since this file isn't being used, instead of commenting out the test bits, you could delete this, or implement the steps in jest to do actual testing of the layout to see that things have been mounted.